docs/user-wallets/wrapped-keys/overview.md

```diff
@@ -0,0 +1,128 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+Wrapped Keys are private keys that are encrypted and stored using Lit's Wrapped Keys backend service. They can be initialized by [importing](./importing-key.md) or [storing](./storing-wrapped-key-metadata.md) an existing private key, or can be [generated](./generating-wrapped-key.md) within a Lit node's trusted execution environment (TEE), meaning the clear text private key never exists outside of a Lit node's TEE.
+
+Regardless of how the Wrapped Key is initialized, each Wrapped Key is associated with an existing [Programmable Keypair (PKP)](../../sdk/wallets/minting.md) and given a unique ID by Lit.
```

Comment:
i think it could be worth expanding on this a bit more. You do a good job distinguishing between PKPs and Wrapped Keys in the intro page, but we never really talk about (until now) the requirement to use both of these together. In this case PKPs are just used to manage permissions / auth on the wrapped key, but this might be unclear / confusing to a new user and could be worth clarifying here